### PR TITLE
Add io.jor.bugdom

### PR DESCRIPTION
**Pull request by keplersj**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** Pending formal approval: https://github.com/jorio/Bugdom/issues/22
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge). (Domain owned upstream, same application used for macOS application bundle)
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* Not relevant to upstream

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by keplersj** [comment]
bot, build io.jor.bugdom

**Comment by keplersj** [comment]
bot, build io.jor.bugdom

File: io.jor.bugdom.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    build-commands:
+      - install -D bugdom_launcher.sh ${FLATPAK_DEST}/bin/Bugdom
+      - mkdir -p ${FLATPAK_DEST}/bugdom
+      - install -D ${FLATPAK_BUILDER_BUILDDIR}/Bugdom ${FLATPAK_DEST}/bugdom/Bugdom
+      - cp -r ./Data ${FLATPAK_DEST}/bugdom
+      - install -D ./packaging/bugdom.desktop ${FLATPAK_DEST}/share/applications/bugdom.desktop
+      - install -D ./packaging/bugdom-desktopicon.png ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps/bugdom-desktopicon.png
+      - install -D ./io.jor.bugdom.metainfo.xml ${FLATPAK_DEST}/share/metainfo/io.jor.bugdom.metainfo.xml
+
+finish-args:
```
Comment (barthalion) [inline io.jor.bugdom.yml:33]: Nitpick: by convention, `finish-args` usually is before `modules`.

Comment (hfiguiere) [reply io.jor.bugdom.yml:33]: it's also mssing --share=ipc

**Comment by pizzadude** [comment]
https://github.com/flathub/io.jor.bugdom/pull/1

**Pull request closed without merging**